tests/e2e/e2e_test.go

```diff
@@ -451,46 +451,59 @@ func TestE2E(t *testing.T) {
 			},
 		},
 		testExecution{
-			Name: "TestScanProducesRemediations",
-			// NOTE(jaosorior): This was made a serial test because it runs the long-running, resource-taking and
-			// big AF moderate profile
-			IsParallel: false,
+			Name:       "TestScanProducesRemediations",
+			IsParallel: true,
 			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
-				scanName := getObjNameFromTest(t)
-				selectWorkers := map[string]string{
-					"node-role.kubernetes.io/worker": "",
+				bindingName := getObjNameFromTest(t)
+				tpName := getObjNameFromTest(t)
+
+				tp := &compv1alpha1.TailoredProfile{
```

Comment:
Can you add a comment explaining why we're adding a TP that only extends and doesn't enable or disable any rules?

Reply:
(And remind me why we're doing that, I know you told me some time ago but I forgot...)

Reply:
@jhrozek done. Mostly it's just us wanting the scans to be unique and for this not to re-use names. This is not a problem in usual deployments, but mostly for our test cases.